### DISCUSSION TITLE
The default convention documentation is misleading

### DISCUSSION THREAD

**Discussion by baruchiro** [semantic-release/semantic-release#2820] [Commit Convention] [answered]
I see the [documentation](https://github.com/semantic-release/semantic-release/blob/master/README.md#commit-message-format) declares
> By default, semantic-release uses [Angular Commit Message Conventions](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-format)

More than that, they are suggesting using the tools:
> Tools such as [commitizen](https://github.com/commitizen/cz-cli) or [commitlint](https://github.com/conventional-changelog/commitlint) can be used to help contributors and enforce valid commit messages.

But the table below there, includes only a few prefixes (`fix|feat|perf`) and some prefixes from Angular are missing (`build|ci|docs|feat|fix|perf|refactor|test`).

First, it is confusing, I needed to merge a few Pull Requests to find out something didn't work as expected.
More than that, I used `commitlint` to verify the commits, and it didn't find any problem.

My questions are:
1. Am I the only one who finds the documentation confusing?
1. How can I easily verify that `semantic-release` and `commitlint` work in the exact same format?

**Comment by travi** [comment, marked as answer]
build, ci, docs, refactor, test should not include changes that impact consumers, so there is no reason to publish a new release since there is no impact to communicate through a bumped semantic version. those commit types are still part of the convention and have value to communicate the type of change within the project itself, but simply do not result in releasing a new version.

if you believe that the docs could be improved, please feel free to suggest changes through a PR and further discussion could take place there.
